test/common/globalBefore.js

```diff
@@ -61,7 +61,7 @@ function waitUntilBlockchainReady (cb, retries, timeout, baseUrl) {
 
 	baseUrl = baseUrl || 'http://' + config.address + ':' + config.httpPort;
 	(function fetchBlockchainStatus () {
-		popsicle.get(baseUrl + '/api/loader/status')
+		popsicle.get(baseUrl + '/api/node/status')
```

Comment:
Please check `test/functional/http/get/node.js` and use the same pattern to make the http calls. That will make the use of swagger based apis consistent. For reference, here is how we can use it. 

```
var swaggerEndpoint = require('../../../common/swaggerSpec');
var nodeStatusEndpoint = swaggerEndpoint('GET /node/status 200');
nodeStatusEndpoint.makeRequest().then(function(res){ });
```

Reply:
That's not in the scope of this PR. @karmacoma should I follow that suggestion?

Reply:
I guess using same pattern we are going to use for all over swagger endpoints is a consistency thing, not related to scope of the PR.

Reply:
Agree. From now on, please use new swagger based API from for consistency.